generator/templates/schemavalidator.gotmpl

```diff
@@ -247,6 +247,18 @@ func ({{ .ReceiverName }} *{{ if .IsExported }}{{ pascalize .Name}}{{ else }}{{
 // Validate validates this {{ humanize .Name }}
 func ({{.ReceiverName}} {{ if or .IsTuple .IsComplexObject .IsAdditionalProperties }}*{{ end }}{{ if .IsExported }}{{ pascalize .Name}}{{ else }}{{ .Name }}{{ end }}) Validate(formats strfmt.Registry) error {
   var res []error
+
+  {{ if .IsPrimitive }}
+    var propName string
```

Comment:
wouldn't you be able to set the prop name just to {{ pascalize .Name }} and not have a runtime component to this?